src/libs/NextStepUtils.ts

```diff
@@ -181,7 +184,9 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
                         [CONST.POLICY.AUTO_REPORTING_FREQUENCIES.IMMEDIATE]: 'later today',
                         [CONST.POLICY.AUTO_REPORTING_FREQUENCIES.WEEKLY]: 'on Sunday',
                         [CONST.POLICY.AUTO_REPORTING_FREQUENCIES.SEMI_MONTHLY]: 'on the 1st and 16th of each month',
-                        [CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MONTHLY]: formattedDate ? `on the ${formattedDate} of each month` : '',
+                        [CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MONTHLY]: formattedDate
+                            ? `on the ${formattedDate} of ${autoSubmissionMonth ? `the ${autoSubmissionMonth}` : 'each month'}`
```

Comment:
`of the February` doesn't look good to me, this should say `of February`

<img width="392" alt="image" src="https://github.com/user-attachments/assets/f3ca602b-b469-432e-a16c-e95196951c98" />